### ISSUE TITLE
Milken and Haas overlap on sponsor slideshow

### ISSUE THREAD

**Issue by jaredhasenklein** [themilkenknights/milkenknightsdotcom#19]

**Comment by jaredhasenklein** [comment]
The slideshow also just looks like crap

**Comment by jaredhasenklein** [comment]
Will become obsolete after #28 is merged. Will create a new PR after #28 is working to remove the slideshow entirely.

**Comment by alexanderswerdlow** [comment]
Fixed with f466a660d9c7d80caafdf95b7f20df8c0e4c612f and 4a594210b76d108e8e9e92ad8f5cab15071f491a
